client/src/components/DataFiles/DataFilesModals/DataFilesPreviewModal.jsx

```diff
@@ -101,7 +101,7 @@ const DataFilesPreviewModal = () => {
           <NiiVue imageUrl={href} fileName={params.path}></NiiVue>
         )}
         {previewUsingHref && !previewUsingBrainmap && (
-          <div className="ratio ratio-4x3">
+          <div className="embed-responsive embed-responsive-4by3">
```

Comment:
Is this a relevant change?

Reply:
This was to undo the change that had happened most likely when I pointed this branch to the Compress branch instead of the Main branch. I think we had briefly talked about it yesterday.

Reply:
I see. Maybe try to make sure that `task/WP-724` has the correct value, then this diff should have no change here

Reply:
All fixed!